packages/paste-design-tokens/tokens/aliases/color-palette.yml

```
@@ -64,6 +70,10 @@ aliases:
   palette-gray-100-transparent-90: "rgba(18, 28, 45, 0.9)"
   palette-gray-100-transparent-100: "rgba(18, 28, 45, 1)"
 
+  ## Grays transparent based on darkest gray
+  palette-gray-110-transparent-05: "rgba(15,22,33 0.05)"
+  palette-gray-110-transparent-10: "rgba(15,22,33 0.1)"
```

Are these missing the last comma and not formatted?

```suggestion
  palette-gray-110-transparent-05: "rgba(15, 22, 33, 0.05)"
  palette-gray-110-transparent-10: "rgba(15, 22, 33, 0.1)"
```

---

good catch